packages/vapor/scss/components/logo-card.scss

```diff
@@ -3,7 +3,8 @@
     margin: $logo-card-margin;
     padding: $logo-card-padding;
     border: $logo-card-border;
-
+    border-radius: $logo-card-border-radius;
+    box-shadow: $logo-card-box-shadow;
```

Comment:
I think we should wait for UX's confirmation before moving on with the logo card style changes. I'll create a separate ticket for it 👍